addon/webextension/background/senderror.js

```diff
@@ -81,6 +81,16 @@ this.senderror = (function() {
     });
   };
 
+  // Every copy of Screenshots has its own unique moz-extension:// URL.
+  // Raven doesn't correctly group errors with different URLs, so we use the
+  // raven.js `dataCallback` option to turn the UUID into "screenshots-addon"
+  // before sending.
+  // TODO: do this here or somewhere else, though? like in reportError?
+  function normalizeURL(data) {
+    data.request.url = data.request.url.replace(/moz-extension:\/\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/, "resource://screenshots-addon");
```

Comment:
The github URL turns out to be more trouble than it's worth - Sentry infers that the module name is the entire path beyond the domain name, which makes it hard to parse the 'culprit' field. We can experiment with better integration later, though.